dev-packages/application-package/src/application-props.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import { BrowserWindowConstructorOptions } from 'electron';
```

Comment:
`import type { BrowserWindowConstructorOptions } from 'electron'` - to make sure that we don't pull anything in runtime by accident

Reply:
Done 👍